README.md

```diff
@@ -1,2 +1,157 @@
 # cesium-webpack-example
-The minimal recommended setup for an application using Cesium with Webpack.
+
+The minimal recommended setup for an application using [Cesium](https://cesiumjs.org/) with [Webpack](https://webpack.js.org/concepts/).
+
+[![Build Status](https://travis-ci.org/AnalyticalGraphicsInc/cesium-webpack-example.svg?branch=using-custom-loader)](https://travis-ci.org/AnalyticalGraphicsInc/cesium-webpack-example)
+
+### Running this application
+
+	npm install
+	npm start
```

Comment:
For some reason on my machine `npm start` opens a browser but doesn't navigate to the app.  If this is a webpack feature, maybe there's a bug?  If we're doing it ourselves, we should probably fix it.

Reply:
I'm just passing the `open` flag to the webpack dev server. I can just remove it, it sounds like it would be easier than fixing /working around a bug.